scripts/dts/extract/interrupts.py

```diff
@@ -69,10 +69,27 @@ def extract(self, node_path, prop, names, def_label):
                         node_path,
                         lambda alias:
                             '_'.join([str_to_label(alias)] +
-                                     l_cell_prefix + name + l_cell_name),
+                                     l_cell_prefix + l_idx + l_cell_name),
                         full_name,
                         prop_alias)
 
+                    if name:
+                        add_prop_aliases(
+                            node_path,
+                            lambda alias:
+                                '_'.join([str_to_label(alias)] +
+                                         l_cell_prefix + name + l_cell_name),
+                            full_name,
+                            prop_alias)
+                    else:
+                        add_prop_aliases(
+                            node_path,
+                            lambda alias:
+                                '_'.join([str_to_label(alias)] +
+                                         l_cell_prefix + name + l_cell_name),
+                            full_name,
+                            prop_alias)
```

Comment:
The `if` and `else` blocks seem to be identical.

Reply:
They are, its so in a follow on commit we'll mark one of them deprecated.